src/colorizer/types.ts

```diff
@@ -32,7 +32,7 @@ export type FeatureArrayType = {
 
 type FeatureTypeSpec<T extends FeatureDataType> = {
   /** The constructor for a `TypedArray` of this numeric type */
-  arrayConstructor: { new (arr: number[]): FeatureArrayType[T] };
+  ArrayConstructor: { new (arr: number[]): FeatureArrayType[T] };
```

Comment:
why caps here?

Reply:
This is an eslint rule (https://eslint.org/docs/latest/rules/new-cap#capisnew) that enforces capitalization on any constructor called with the new keyword. This array constructor is usually called like this:
```
const buffer = new spec.ArrayConstructor(data);
```

I can modify the eslint configuration so it doesn't flag properties (i.e., `new spec.arrayConstructor` wouldn't trigger an error), if that makes sense to you!

Reply:
Thanks for the explanation!  Makes sense to me.  No need to change anything.

Reply:
Was unsure when I wrote this whether I ought to capitalize it like any other property of `FeatureTypeSpec` or like a `TypedArray` constructor. Glad to hear eslint has a definitive opinion on this.